packages/ui/src/bounty/components/TileSection.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+import { MemoryRouter } from 'react-router'
+
+import { TileSection, TileSectionProps } from '@/bounty/components/TileSection'
+import { MockApolloProvider } from '@/mocks/components/storybook/MockApolloProvider'
+
+export default {
+  title: 'Bounty/TileSection',
+  component: TileSection,
+} as Meta
+
+const Template: Story<TileSectionProps> = (args) => (
+  <MemoryRouter>
+    <MockApolloProvider>
+      <TileSection {...args} />
+    </MockApolloProvider>
+  </MemoryRouter>
+)
+
+export const Default = Template.bind({})
+Default.args = {
```

Comment:
Could you please add a story for each period? And rename Default to Funding :)

Reply:
ok, i created all periods variant